feat: add MultiLoRA loader node with custom changes from Coziness repo

meklithab opened the pull request:
- Added MultiLoRALoader node based on the Coziness repository, with custom modifications.

- Updated the node’s input types to match the output format of the existing AutoLoraLoader_S2V node for seamless integration.

- Preserves existing functionality for stacking multiple LoRAs from both text input and incoming Lora stacks.

- Integrated the node into __init__.py with proper mappings for class and display name.

meklithab committed:
feat: add MultiLoRA loader node with custom changes from Coziness repo

bisratberhanu commented on comfyui_script_to_video_suite/s2v_nodes/MultiLoraLoader.py:
```diff
@@ -0,0 +1,211 @@
+                              "lora_stack": ("LORA_STACK",),  # optional input
+
+                            }}
+
```
Why is lora stack an optional field in the first place, it should be  a required field because that is the only viable way of  giving input. This also means the text field is not required. Remove it

bisratberhanu commented on comfyui_script_to_video_suite/s2v_nodes/MultiLoraLoader.py:
```diff
@@ -0,0 +1,211 @@
+NODE_DISPLAY_NAME_MAPPINGS = {
+    "MultiLoraLoader-70bf3d77": "MultiLora Loader",
+    "LoraTextExtractor-b1f83aa2": "Lora Text Extractor",
+}
```
you have already imported the files in the `__init__.py` file so they are not required  here.

bisratberhanu commented on comfyui_script_to_video_suite/s2v_nodes/MultiLoraLoader.py:
```diff
@@ -0,0 +1,211 @@
+        # 1️⃣ Apply loras from incoming stack if available
+        if lora_stack:
+            for lora_path, strength_model, strength_clip in lora_stack:
+                lora_sd = comfy.utils.load_torch_file(lora_path)
```
Very important, make sure things are working.

meklithab committed:
refactor(nodes): simplify MultiLoraLoader into MultiLoraLoader_S2V with caching and streamlined inputs

- Removed text parsing logic, SelectedLoras, LoraItemsParser, and LoraTextExtractor classes
- Dropped dependency on regex-based prompt parsing and inline LoRA extraction
- Introduced MultiLoraLoader_S2V class focused solely on applying LoRA stacks
- Added path resolution via folder_paths for relative LoRA names
- Implemented caching to avoid reloading identical LoRA files multiple times
- Added validation and error handling for LoRA strength values
- Updated category to "Script To Video Suite" for better organization
